exercises/07-normalizr/backend/server.ts

```diff
@@ -0,0 +1,77 @@
+import express from 'express'
+
+import { User, UserData } from '../src/modules/users/user-types'
+
+const port = 3001
+
+const areStringsSame = (a: string, b: string): boolean =>
+  a.toLowerCase() === b.toLowerCase()
+
+const hasSameName = (
+  user: UserData,
+  firstName: string,
+  lastName: string
+): boolean =>
+  areStringsSame(firstName, user.firstName) &&
+  areStringsSame(lastName, user.lastName)
+
+const findRegnalNumber = (users: User[], userName: UserData) => {
```

Comment:
regional?

Reply:
It's regnal, as in https://en.wikipedia.org/wiki/Regnal_number